parcel_test.go

```diff
@@ -31,57 +32,127 @@ func getTestParcel() Parcel {
 // TestAddGetDelete проверяет добавление, получение и удаление посылки
 func TestAddGetDelete(t *testing.T) {
 	// prepare
-	db, err := // настройте подключение к БД
+	db, err := sql.Open("sqlite", "tracker.db")
+	require.NoError(t, err)
+	defer db.Close()
 	store := NewParcelStore(db)
 	parcel := getTestParcel()
 
 	// add
-	// добавьте новую посылку в БД, убедитесь в отсутствии ошибки и наличии идентификатора
+	id, err := store.Add(parcel)
+
+	// Если произошла ошибка при добавлении Parsel, следующие тесты будут бесполезны
+	require.NoError(t, err)
+	require.NotEmpty(t, id)
 
 	// get
 	// получите только что добавленную посылку, убедитесь в отсутствии ошибки
-	// проверьте, что значения всех полей в полученном объекте совпадают со значениями полей в переменной parcel
+	actual, err := store.Get(id)
+
+	// // Если произошла ошибка, дальнейшие тесты бесполезны
+	require.NoError(t, err)
 
+	// // Если не равны, есть вероятность ложных результатов следующих тестов
+
+	require.Equal(t, id, actual.Number)
+	// проверьте, что значения всех полей в полученном объекте совпадают со значениями полей в переменной parcel
+	assert.Equal(t, parcel.Client, actual.Client)
+	assert.Equal(t, parcel.Status, actual.Status)
+	assert.Equal(t, parcel.Address, actual.Address)
+	assert.Equal(t, parcel.CreatedAt, actual.CreatedAt)
 	// delete
 	// удалите добавленную посылку, убедитесь в отсутствии ошибки
-	// проверьте, что посылку больше нельзя получить из БД
+	err = store.Delete(id)
+	require.NoError(t, err)
+
+	// Проверка, что посылку больше нельзя получить из БД
+	actual, err = store.Get(id)
+	require.Empty(t, actual.Number)
+	require.Equal(t, sql.ErrNoRows, err)
```

Comment:
Для сравнивания ошибок нужно использовать специализированную функцию
```suggestion
	require.ErrorIs(t, err, sql.ErrNoRows)
```